wagtail/admin/userbar.py

```diff
@@ -85,6 +86,14 @@ def __init__(self, in_editor=False):
             # If omitted, defaults to True and overrides configs in `axe_run_only`.
             "enabled": True,
         },
+        {
+            "id": "meta-description",
+            "impact": "moderate",
+            "selector": "html",
```

Comment:
The selector shown and used is `html` because the other selectors like `head`/`meta`/`meta[name="description"]` doesn't work.